### ISSUE TITLE
Godot is randomly unable to load the default brush on startup

### ISSUE THREAD

**Issue by Zylann** [Zylann/godot_heightmap_plugin#219]
**Describe the bug**
Rarely, but randomly, Godot fails to load the default terrain brush:
```
ERROR: load_image: Error opening file 'addons/zylann.hterrain/tools/brush/shapes/round2.exr'.
   At: core/io/image_loader.cpp:56
ERROR: call: brush_editor.gd: Could not load image at 'addons/zylann.hterrain/tools/brush/shapes/round2.exr', error [7]: File Not found error
   At: modules/gdscript/gdscript_functions.cpp:803
```
Workaround: manually clicking on the brush in the bottom panel and selecting `round2.exr` fixes it.

**To Reproduce**
Happens randomly upon launch, when the plugin is already enabled in project settings, even if it was already imported in a previous session.

**Expected behavior**
The brush should load fine. The file is present and should be able to load.

**Environment**
- OS: Windows 10
- Godot version: 3.2.3
- Plugin version: 1.5 dev
- Renderer used: GLES3

**Comment by Sslaxx** [comment]
Is this reported on the Godot issuetracker? If so would be useful to know what issues there are related to this one.

**Comment by Zylann** [comment]
I don't think it was reported, I didnt do it because I have no reliable reproduction.

**Comment by Zylann** [comment]
I just had this happen with a regular PNG while choosing an image in the import tool. Trying to load it again worked. There is something fishy going on, I still don't know how to reproduce it.
